app/controllers/pto_requests_controller.rb

```diff
@@ -126,12 +171,11 @@ def destroy
             if @pto_request.reason == 'make up / sick day'
                 return sub_make_up_day
             end 
-
             remove_request_info
             RequestsMailer.with(user: @user, pto_request: @pto_request).delete_request_email.deliver_now
             return redirect_to root_path, notice: "Your request was deleted"
```

Comment:
redirecting to the root_path is kinda jarring, not expecting to go back to the calendar after deleting a user's request.

Reply:
should be resolved by 05f8385